forui/lib/src/widgets/select_menu_tile.dart

```diff
@@ -27,6 +30,34 @@ class FSelectMenuTile<T> extends FormField<Set<T>> with FTileMixin {
   /// The controller that shows and hides the menu. It initially hides the menu.
   final FPopoverController? popoverController;
 
+  /// The scroll controller used to control the position to which this menu is scrolled.
+  ///
+  /// Scrolling past the end of the group using the controller will result in undefined behaviour.
+  ///
+  /// It is ignored if the group is part of a merged [FTileGroup].
+  final ScrollController? scrollController;
+
+  /// The cache extent in logical pixels.
+  ///
+  /// Items that fall in this cache area are laid out even though they are not (yet) visible on screen. It describes
+  /// how many pixels the cache area extends before the leading edge and after the trailing edge of the viewport.
+  ///
+  /// It is ignored if the group is part of a merged [FTileGroup].
+  final double? cacheExtent;
+
+  /// The max height, in logical pixels. Defaults to infinity.
+  ///
+  /// It is ignored if the group is part of a merged [FTileGroup].
```

Comment:
I don't quite understand this part of the documentation. Isn't this for the popover, what has that got to do with `FTileGroup`?